docs/UserGuide.adoc

```diff
@@ -219,6 +219,8 @@ Displays the userguide from within the app.
 
 Format: `help`
 
+<<Features,Not the feature you are looking for? Check out _FreeTime_'s other features>>
```

Comment:
After every feature clicking this link brings the user back to the top, where they can use the table to navigate down again, increasing navigability.

But it might seem to make the document slightly cluttered, I'd like to hear from you guys @bennchong @nianfei97 @alexiscatnip

Reply:
![image](https://user-images.githubusercontent.com/25686312/48311548-3637ea00-e5dc-11e8-9908-585fc71c6ee4.png)

I think its really nifty to have that link thanks! But one suggestion is to make it perhaps shorter, such as 'Jump back to feature list' or something along those lines.

Reply:
I'll wait for one more person's opinion than we can merge